src/main/java/quotes/Quote.java

```diff
@@ -3,12 +3,14 @@
 public class Quote {
    public String author;
    public String text;
+   public String starWarsQuote;
 
 
 //  Constructor
-   public Quote (String author, String text){
+   public Quote (String author, String text, String starWarsQuote){
```

Comment:
Could you get away with just inputting the quote's text regardless of whether or not it's a quote from the file or from Star Wars?